scoper.inc.php

```diff
@@ -109,6 +110,9 @@ function( $file_path, $prefix, $contents ) {
 				$contents = str_replace( "'Google_", "'" . $prefix . '\Google_', $contents );
 				$contents = str_replace( '"Google_', '"' . $prefix . '\Google_', $contents );
 			}
+			if ( preg_match( '#vendor/symfony/polyfill-.*/bootstrap.php$#', $file_path ) ) {
+				$contents = str_replace( "namespace $prefix;", "/* namespace $prefix intentionally removed */", $contents );
+			}
```

Comment:
I've noticed the E2E tests have started failing since introducing this block. I tried running the first failing spec file, `tests/e2e/specs/modules/tagmanager/setup.test.js`, locally and got this error in the PHP error log:

```
[07-Mar-2023 17:02:11 UTC] PHP Fatal error:  Uncaught Error: Class 'Normalizer' not found in /var/www/html/wp-content/plugins/google-site-kit/third-party/symfony/polyfill-intl-idn/Idn.php:242
Stack trace:
#0 /var/www/html/wp-content/plugins/google-site-kit/third-party/symfony/polyfill-intl-idn/Idn.php(149): Google\Site_Kit_Dependencies\Symfony\Polyfill\Intl\Idn\Idn::process('localhost', Array, Object(Google\Site_Kit_Dependencies\Symfony\Polyfill\Intl\Idn\Info))
#1 /var/www/html/wp-content/plugins/google-site-kit/third-party/symfony/polyfill-intl-idn/bootstrap.php(148): Google\Site_Kit_Dependencies\Symfony\Polyfill\Intl\Idn\Idn::idn_to_utf8('localhost', 0, 1, Array)
#2 /var/www/html/wp-content/plugins/amp/includes/class-amp-http.php(247): idn_to_utf8('localhost', 0, 1)
#3 /var/www/html/wp-content/plugins/amp/includes/class-amp-http.php(205): AMP_HTTP::get_amp_cache_hosts()
#4 /var/www/html/wp-includes/class-wp-hook.php(310): AMP_HTTP::filter_allowed_redirect_hosts(Array)
#5 /var/www/html/wp-includes/plugin.php(205): WP_Hook->apply_filters(Array, Array)
# in /var/www/html/wp-content/plugins/google-site-kit/third-party/symfony/polyfill-intl-idn/Idn.php on line 242
```

Looking at `third-party/symfony/polyfill-intl-idn/Idn.php`, here's how it's trying to use `Normalizer`:

```
// third-party/symfony/polyfill-intl-idn/Idn.php:

namespace Google\Site_Kit_Dependencies\Symfony\Polyfill\Intl\Idn;

/* ... */

use Normalizer;

/* ... */

        if (!\Normalizer::isNormalized($domain, \Normalizer::FORM_C)) {
                $domain = \Normalizer::normalize($domain, \Normalizer::FORM_C);
        } 
```

Noting the polyfilled version of `Normalizer` provided in `third-party/symfony/polyfill-intl-normalizer/Normalizer.php`:
```
// third-party/symfony/polyfill-intl-normalizer/Normalizer.php:

namespace Google\Site_Kit_Dependencies\Symfony\Polyfill\Intl\Normalizer;

/* ... */

class Normalizer
{
```

... I tried patching `Google\Site_Kit_Dependencies\Symfony\Polyfill\Intl\Idn` to use this polyfilled version, but it still couldn't find it.

I didn't want to spend more time digging into a potential rabbit hole, so left it there. Interested to know if you have thoughts on a fix. I notice there are a few references to `\Normalizer` within `third-party/symfony/polyfill-*` ...

Reply:
Oh strange, I'll have a look – thanks! Odd that the E2E tests don't show as failed for me but cancelled 🤔

Reply:
> Oh strange, I'll have a look – thanks! Odd that the E2E tests don't show as failed for me but cancelled :thinking:

Oh, well yes - to be clear, they do show as cancelled for me too, on the PR, but as a result of timing out due to test failures. On the Actions tab for the workflow the recent runs do show up as failures, e.g. [here](https://github.com/google/site-kit-wp/actions/workflows/e2e-tests-wp-latest.yml?query=branch%3Aenhance%2F1146-guzzle6).

![image](https://user-images.githubusercontent.com/18395600/223998165-defac468-9e53-4b9e-bc50-1bae499e702c.png)